plugins/generator-1.18.1/forge-1.18.1/procedures/shoot_arrow.java.ftl

```diff
@@ -0,0 +1,11 @@
+if (${input$entity} instanceof LivingEntity _ent_sa && !_ent_sa.level.isClientSide()) {

```

Comment:
Maybe you should add space between `if` and braces in other files too?

Reply:
I didn't change files. They are the exact same files as files used in Forge 1.17.1.